docs/components/camera/velodyne.md

```diff
@@ -42,12 +42,12 @@ Fill in the attributes for your velodyne camera:
 {{% /tab %}}
 {{< /tabs >}}
 
-The following attributes are available for velodyne cameras:
+The following attributes are available for `velodyne` cameras:
 
 | Name | Inclusion | Description |
 | ---- | --------- | ----------- |
-| `port` | *Required* | Specify the port the velodyne camera is running on. |
-| `ttl_ms` | *Required* | Specify the  milliseconds ???. |
+| `port` | **Required** | Port the velodyne camera is running on. |
```

Comment:
```suggestion
| `port` | **Required** | The port the Velodyne camera is running on. |
```
"The" is [opt]. Capital is not opt. I'm thinking the "The" helps the reader immediately interpret "port" as a noun instead of as a verb.